### ISSUE TITLE
About the stability & params

### ISSUE THREAD

**Issue by banianzr** [lucidrains/stylegan2-pytorch#123]
Hi, thanks for the great works! I'm new to GAN and I tried to use it on my dataset with 890 pics. During my experiment, I tested some parameters and I found some problems:

1. I repeated the experiment with same parameters, but the results varied. I'm wondering whether it is stable? For example, with param:
```
--batch-size 6 \
--gradient-accumulate-every 4 \
--attn-layers [1,2] \
--aug-prob .25 \
--dataset-aug-prob .5 \
--cl-reg
```

I got results as below:

- Experiment 1:

	- overview

	![exp-1](https://user-images.githubusercontent.com/5343481/92109677-0a729d80-ee1c-11ea-8bce-be17429188f3.png)

	- Iter 2k:

	![2-mr](https://user-images.githubusercontent.com/5343481/92109808-43127700-ee1c-11ea-9388-acd60a89b546.jpg)

	- Iter 21k:

	![21-mr](https://user-images.githubusercontent.com/5343481/92109942-75bc6f80-ee1c-11ea-83f2-aa16e992be27.jpg)

- Experiment 2:

	- overview

	![exp-2](https://user-images.githubusercontent.com/5343481/92109682-0cd4f780-ee1c-11ea-9273-7e3364efb61e.png)

	- Iter 2k:

	![2-mr](https://user-images.githubusercontent.com/5343481/92109986-8bca3000-ee1c-11ea-8127-11086ccb62ec.jpg)

- Experiment 3:

	- overview

	![exp-3](https://user-images.githubusercontent.com/5343481/92109689-0d6d8e00-ee1c-11ea-98f9-607f070bb850.png)

	- Iter 2k:

	![2-mr](https://user-images.githubusercontent.com/5343481/92110137-d8157000-ee1c-11ea-9eef-d4613e9721f4.jpg)

	- Iter 20k:

	![20-mr](https://user-images.githubusercontent.com/5343481/92110272-08f5a500-ee1d-11ea-8c28-5b86e098baad.jpg)

2. About memory considerations, is there any constraint between **batch size** and **network capacity**?

Thanks in advance!!!

**Comment by lucidrains** [comment]
@banianzr Hello! I've updated the differentiable augmentations to include translation and color, would you like to try again and let me know how it goes?

**Comment by banianzr** [comment]
@lucidrains Hi, sorry for the late response! 
During the past days, I've done some experiments and recorded 2 groups of feasible arguments based on the previous version. Results are pasted below:
![0904](https://user-images.githubusercontent.com/5343481/94508585-d89afe00-0244-11eb-86e9-3e8d9c0e41d3.png)
iter 77k for `--batch-size 7 --attn-layers [1,2] --network-capacity 24 --aug-prob .25 --dataset-aug-prob .5`
![0905](https://user-images.githubusercontent.com/5343481/94508601-df297580-0244-11eb-9854-35779a86b884.png)
iter 83k for `--aug-prob .25 --dataset-aug-prob .5`

It is a dataset of caterpillar, and even after 150k rounds, the results is almost the same as that at iter 80k.
Also in my case, the `--cl-reg` may lead to memory overflow thus I didn't use that.

I also trained on a snake dataset, but I won't put it here because it might cause discomfort. The result is worse than caterpillar dataset. Maybe it's because the proportion of snakes in the image is too small, and there are too many shapes?

I'll try the new version in the coming days, and I also noticed that you add multi-GPU training! Great thanks for that!!!
